src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
+    public String getRandomBall() {
+        Random random = new Random();
```

Comment:
Creating a new instance of 'Random' every time the method is called is inefficient. Consider making 'random' a private final field of the class.